server/utils/utils.ts

```diff
@@ -185,6 +185,8 @@ export function isMandatoryTextValue(val: unknown): boolean {
 
 export const isEmptyStringOrWhitespace = (val: string | string[]) => !val || !(val as string).trim()
 
+export const isNullEmptyStringOrWhitespace = (text: string) => text == null || text.match(/^\s*$/) !== null
+
```

Comment:
I don't think this is any different in functionality to the isEmptyStringOrWhitespace on the line above.

Reply:
ah ok! Good catch! Corrected!